dist/bundle.js

```diff
@@ -286,6 +286,17 @@ return /******/ (function(modules) { // webpackBootstrap
 	    }
 	  };
 
+	  function _update(endpoint, options) {
+	    _validateEndpoint(endpoint);
+	    optionValidators.data(options);
+	    var ref = new Firebase(baseUrl + '/' + endpoint);
```

Comment:
@foffer  can you change line 292 to be compatible with the new firebase 3 release? 
```
var ref = firebase.database().ref(endpoint);
```